libs/components/ui/src/model/index.tsx

```diff
@@ -32,11 +32,13 @@ export const TransitionsModal = (props: TransitionsModalProps) => {
             onClose={props.onClose}
             closeAfterTransition
         >
-            <MuiClickAwayListener onClickAway={props.onClose}>
-                <Fade in={props.open} timeout={300}>
-                    {props.children}
-                </Fade>
-            </MuiClickAwayListener>
+            <div>
```

Comment:
What is the purpose of the new div?

Reply:
![image](https://user-images.githubusercontent.com/21084335/194989841-b5d885aa-321d-41af-86b6-1ff6c132e70c.png)